test/sanity/run-test.sh

```diff
@@ -31,7 +31,11 @@ if [[ "$#" -gt 0 ]] && [[ -n "$1" ]]; then
   nodeid="$1"
 fi
 
-_output/azurediskplugin --endpoint "$endpoint" --nodeid "$nodeid" -v=5 &
+if [[ "$#" -lt 2 || "$2" != "v2" ]]; then
```

Comment:
should add a new `pull-azuredisk-csi-driver-sanity-v2` test pipeline in test-infra first, btw, I doubt whether v2 driver would run sanity test in the end(it could now since code is the same as v1) since v2 driver depends on CRD, while sanity test does not run inside a k8s cluster.

Reply:
cc @mukhoakash

Reply:
I separated the v1 & v2 sanity & integration test targets again. I submitted [kubernetes/test-infra PR#20981](https://github.com/kubernetes/test-infra/pull/20981) to add the new pipelines.

@andyzhangx Agreed that we'll have challenges sanity testing the v2 driver with the CRDs. We'll need to give some thought around how we can do that. I'd really hate to lose the coverage.